.jshintrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Would be good to populate the jshint (e.g. https://github.com/fheng/fh-supercore/blob/master/.jshintrc)